tests/unit/test_charm.py

```diff
@@ -510,17 +579,25 @@ def test_initialise_replica_failure_leads_to_waiting_state(
             )
 
     @patch_network_get(private_address="1.1.1.1")
-    @patch("mongod_helpers.MongoDB.is_ready")
+    @patch("mongod_helpers.MongoDB.is_replica_ready")
+    @patch("mongod_helpers.MongoDB.is_mongod_ready")
     @patch("mongod_helpers.MongoDB.initialise_replica_set")
     @patch("charm.MongodbOperatorCharm._open_port_tcp")
     @patch("charm.service_running")
     @patch("mongod_helpers.MongoDB.is_replica_set")
     def test_initialise_replica_success(
-        self, is_replica, service_running, _open_port_tcp, initialise_replica_set, is_ready
+        self,
+        is_replica,
+        service_running,
+        _open_port_tcp,
+        initialise_replica_set,
+        is_mongod_ready,
+        is_replica_ready,
     ):
         self.harness.set_leader(True)
         service_running.return_value = True
-        is_ready.return_value = True
+        is_mongod_ready.return_value = True
+        is_replica_ready.return_value = True
```

Comment:
When you're patching a method, `return_value` can be set in the `@patch` statements, making these lines unnecessary for simple variables like booleans.